rcsb/db/mongo/PdbxLoader.py

```diff
@@ -639,6 +639,13 @@ def load(
 
             collectionNameList = collectionLoadList if collectionLoadList else fullCollectionNameList
 
+            # Move "entry" collection to the end of the list so that if it fails midload, we can determine which entities/assemblies/etc. need reloading based on entry collection
+            colL = collectionNameList
```

Comment:
I don't think this is actually creating a new list, unless Python behaves differently than other OOP languages. So I think it's still modifying the list while looping through it.

Maybe use list comprehension to create a copied list, or make two lists (one with `core_entry` items and one with the rest) and then concatenate them in the desired order.

Reply:
Yes that's correct, it's just a linker to the list. It's weird to me that Python was OK with this though, since if you try to iterate over `collectionNameList` and modify it in place, Python complains. But somehow it lets you do this. Not sure why.

Anyway, yes the list comprehension is a better idea, thanks.

Reply:
I take that back, it doesn't complain about that either. I remember seeing something complain about that before though. Maybe it was Pylint.

Reply:
Makes sense that it could have been pylint or an IDE feature or something.